.chloggen/add_inspect_metrics.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: dockerstatsreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Add [container.cpu.limit], [container.cpu.shares] and [container.restarts] metrics from docker container api"
+
+# One or more tracking issues related to the change
+issues: [21087]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: This is a breaking change as it raises the minimum API version to 1.25
```

Comment:
Can we make this a non-breaking change where we would only mandate version 1.25 if one of the metrics we are adding are enabled?

Reply:
Updated here: [58b5270](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/21803/commits/58b5270435394ba7bafeff0b0abd844ebf76ecf3)